presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/thrift/ThriftHiveMetastore.java

```diff
@@ -133,17 +174,28 @@ public ThriftHiveMetastoreStats getStats()
         return stats;
     }
 
+    @Managed
+    @Flatten
+    public List<ThriftHiveMetastoreStats> getAllStats()
+    {
+        return new ArrayList<>(hmsStats.values());
+    }
+
+    private RetryDriver retry()
```

Comment:
do we need to move retry() and propagate() here? Or just leave them there

Reply:
I can move it back no problem